packages/file-explorer/src/FileListContainer.tsx

```diff
@@ -191,7 +191,7 @@ export function FileListContainer(props: FileListContainerProps): JSX.Element {
     (itemProps: FileListRenderItemProps): JSX.Element => {
       const { item } = itemProps;
       if (renameItem && renameItem.filename === item.filename) {
-        return renderFileListItem({
+        return RenderFileListItem({
```

Comment:
Any reason not to write this part like:
```
  const renderItem = useCallback(
    (itemProps: FileListRenderItemProps): JSX.Element => {
      const { item } = itemProps;
      if (renameItem && renameItem.filename === item.filename) {
        return (
          <RenderFileListItem {...itemProps}>
            <FileListItemEditor
              item={item}
              validate={validateRenameItem}
              onSubmit={handleRenameSubmit}
              onCancel={handleRenameCancel}
            />
          </RenderFileListItem>
        );
      }
      return <RenderFileListItem {...itemProps} />;
    },
    [handleRenameCancel, handleRenameSubmit, renameItem, validateRenameItem]
  );
```

Reply:
Don't think so. I left it initially since I just did minimal required changes to stop the HMR invalidations. I updated to this. `renderFileItemList` was a functional component in disguise